utils/comparable_bytes.cc

```diff
@@ -307,6 +364,32 @@ struct from_comparable_bytes_visitor {
         decode_signed_long_type(comparable_bytes_view, out, true);
     }
 
+    // Decode a byte-comparable representation to the varint number.
+    void operator()(const varint_type_impl&) {
+        // Check the first byte to determine if the bytes were encoded using long or full-form encoding
```

Comment:
Please push to a named function, for readability.

Reply:
Done.